test/check-machines-create

```diff
@@ -410,6 +410,7 @@ class TestMachinesCreate(VirtualMachinesCase):
         # Test create VM with disk of type "block"
         # Check choosing existing volume as destination storage
         runner.createThenInstallTest(TestMachinesCreate.VmDialog(self, sourceType='file',
+                                                                 os_name=config.WINDOWS_SERVER_10,
```

Comment:
Do we now have any test case which checks an iso with Linux?

Reply:
No that would fail now anyway on iso validation - see commit message.

Reply:
I'm confused -- the commit message still talks about "use --location for linux", and that's what the code does AFAIUI, in the [else branch](https://github.com/cockpit-project/cockpit-machines/pull/339/files#diff-1ca34eaf4fc88fdb7fb68685e74a6c1701b7bc2ebb92541110b1b6da8433eb16L171). This should be covered somehow, to make sure that --location works with Linux? (That may require having an iso first, of course -- what you already started to discuss above, like we need to add one to our bots?)